codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/testutils/ExecuteCommand.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.smithy.go.codegen.testutils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Logger;
+import software.amazon.smithy.utils.SmithyBuilder;
+
+/**
+ * Utility for invoking command line utilities.
+ */
+public final class ExecuteCommand {
+    private static final Logger LOGGER = Logger.getLogger(ExecuteCommand.class.getName());
+
+    private final String[] command;
```

Comment:
Not sure if we have it setup, but usually `String[]` will be a spotbugs item. Usually want this to be a `List<String>`.

Reply:
Interesting, Spotbugs doesn't call this out as an issue. Would it be better for this to be a List instead?  Its a array since that is the input the underlying exec method takes.